src/GovUk.Education.ExploreEducationStatistics.Admin/Services/EducationInNumbersService.cs

```diff
@@ -239,13 +239,25 @@ public async Task<Either<ActionResult, EinSummaryViewModel>> PublishPage(
     public async Task<Either<ActionResult, List<EinSummaryViewModel>>> Reorder(
         List<Guid> newOrder)
     {
-        var pageList = await contentDbContext.EducationInNumbersPages
+        var subPageList = await contentDbContext.EducationInNumbersPages
+            .Where(page => page.Slug != null)
             .GroupBy(page => page.Slug)
             .Select(group => group
                 .OrderByDescending(p => p.Version)
                 .First())
             .ToListAsync();
 
+        // GroupBy doesn't play nice with null slugs, hence the need to fetch the root page separately
```

Comment:
Could you not instead force a non-null but obvious default in the GroupBy e.g.:

```
        var pageList = await contentDbContext.EducationInNumbersPages
            .GroupBy(page => page.Slug ?? "ROOT PAGE")
            .Select(group => group
                .OrderByDescending(p => p.Version)
                .First())
            .ToListAsync();
```

Or force it into memory:

```
        var pageList = (await contentDbContext.EducationInNumbersPages
            .ToListAsync())
            .GroupBy(page => page.Slug)
            .Select(group => group
                .OrderByDescending(p => p.Version)
                .First());
```